Modules/src/StatusAggregator.cc

```diff
@@ -116,7 +116,8 @@ namespace ChimeraTK {
   int StatusAggregator::getPriority(StatusOutput::Status status) {
```

Comment:
You can make this function const. Maybe it allows some more optimisation.

Reply:
ok, doesn't hurt